### ISSUE TITLE
Support for SwiftyJSON?

### ISSUE THREAD

**Issue by josephyzhou** [pusher-community/pusher-websocket-swift#25]
Hi guys, 
Are you compatible with SwiftyJSON? For example, can I do:

```swift
pusher.bind("new-comment", callback: { (data: AnyObject?) -> Void in
    if let data = data as? JSON {
       // more stuff
    }
})

```

**Comment by hamchapman** [comment]
Hi @josephyzhou!

You can set the value of `attemptToReturnJSONObject` to false in your Pusher initializer, like so:

```
let pusher = Pusher(
  key: "APP_KEY",
  options: [ "attemptToReturnJSONObject": false ]
)
```

More info about configuration in the docs: https://github.com/pusher-community/pusher-websocket-swift#configuration

Then when you receive the data in the callback it will be a `String` that you can then parse using any JSON parser you'd like, including SwiftyJSON. Specifically, with SwiftyJSON it looks like you'll need to do something like this (a bit pseudo-codey so you might need to tweak it):

```
pusher.bind("new-comment", callback: { (jsonString: AnyObject?) -> Void in
  if let dataFromString = jsonString.dataUsingEncoding(NSUTF8StringEncoding, allowLossyConversion: false) {
    let json = JSON(data: dataFromString)
    // do stuff with your json object
  }
})
```

**Comment by dral3x** [comment]
Hi there,
i notice that the flag you're talking about was not working, so I fixed the issue and made a #63 

Then you can make a JSON object from a String the way you want.
